samza-api/src/main/java/org/apache/samza/operators/MessageStream.java

```diff
@@ -121,14 +121,19 @@
   void sink(SinkFunction<? super M> sinkFn);
 
   /**
-   * Allows sending messages in this {@link MessageStream} to an {@link OutputStream}.
+   * Allows sending messages in this {@link MessageStream} to an {@link OutputStream} and then propagates this
+   * {@link MessageStream} to the next chained operator
    * <p>
    * When sending messages to an {@code OutputStream<KV<K, V>>}, messages are partitioned using their serialized key.
    * When sending messages to any other {@code OutputStream<M>}, messages are partitioned using a null partition key.
-   *
+   * <p>
+   *  Note: The message will be written but not flushed to the underlying output system before its propagated to the
```

Comment:
Nitpick: Fix indentation (off by 1). Don't need the last <p> (line 132). Same for the method below.